lib/workers/repository/init/merge.ts

```diff
@@ -179,13 +180,13 @@ export async function mergeRenovateConfig(
   }
   checkForRepoConfigError(repoConfig);
   const migratedConfig = await migrateAndValidate(config, configFileParsed);
-  if (migratedConfig.errors.length) {
+  if (migratedConfig.errors?.length) {
     const error = new Error(CONFIG_VALIDATION);
     error.validationSource = repoConfig.configFileName;
     error.validationError =
       'The renovate configuration file contains some invalid settings';
     error.validationMessage = migratedConfig.errors
-      .map((e) => e.message)
+      ?.map((e) => e.message)
```

Comment:
```suggestion
      .map((e) => e.message)
```

not required, as already validated by condition